tools/common/schema/updatetask.go

```diff
@@ -55,12 +55,12 @@ type (
 		md5                  string
 	}
 
-	// changeSet represents all the changes
+	// ChangeSet represents all the changes
 	// corresponding to a single schema version
-	changeSet struct {
-		version  string
-		manifest *manifest
-		cqlStmts []string
+	ChangeSet struct {
+		Version  string
+		Manifest *manifest
```

Comment:
Is it worth exposing `manifest`?

Reply:
perhaps not, I don't have strong opinions, I can hide it